pyquil/tests/test_device.py

```diff
@@ -74,11 +148,25 @@ def test_kraus_model():
     assert KrausModel.from_dict(d) == km
 
 
-def test_noise_model():
-    km1 = KrausModel('I', (5.,), (0, 1), [np.array([[1 + 1j]])], 1.0)
-    km2 = KrausModel('RX', (np.pi / 2,), (0,), [np.array([[1 + 1j]])], 1.0)
-    nm = NoiseModel("my_qpu", [km1, km2], {0: np.eye(2), 1: np.eye(2)})
+def test_noise_model(kraus_model_I, kraus_model_RX90):
```

Comment:
i think it would be super cool if this test looked like `test_isa` and took in the kraus dicts as input, built the noise model dict from them, then built the `NoiseModel` object, etc.